app/src/main/java/com/xa/xpensauditor/TabFragment.java

```diff
@@ -67,6 +81,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
     }
 
+    /**
+     * Loads data when view is created.
+     * @param view
+     * @param savedInstanceState
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.NonEmptyAtclauseDescriptionCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>At-clause should have a non-empty description.